packages/graphql-docs-generator/src/index.ts

```diff
@@ -1,112 +1,116 @@
-import * as fs from 'fs';
-import * as path from 'path';
-import * as handlebars from 'handlebars';
-import * as prettier from 'prettier';
-const DEFAULT_MAX_DEPTH = 3;
-
+const handlebars = require('handlebars/dist/handlebars');
```

Comment:
`require` instead of `import`?

Reply:
This is a specific file we're importing from that module that is browser compatible. Hence using `require` only works in this case.